modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/SwipeableStars.kt

```diff
@@ -204,11 +207,25 @@ private fun getDesiredStopPoint(
     when (stopPointType) {
         StopPointType.None -> touchX // ignore stop points
 
-        StopPointType.FullAndHalfStars ->
-            stopPoints
-                .minByOrNull { abs(it - touchX) }
-                ?.toFloat()
-                ?: 0f
+        StopPointType.FullAndHalfStars -> {
+            val desiredStopPoint: Float = (
+                stopPoints
+                    .minByOrNull { abs(it - touchX) }
+                    ?.toFloat()
+                    ?: 0f
+                )
+
+            val desiredStarIndex = stopPoints.indexOf(desiredStopPoint.toDouble())
+
+            // Verify if the user is trying to select zero star or a half star
+            if (desiredStarIndex == zeroStarIndexInStopPoints || desiredStarIndex == halfStarIndexInStopPoints) {
+                // If trying to select zero star or a half star, we will select one full star instead
+                stopPoints.getOrNull(oneFullStarIndexInStopPoints)?.toFloat() ?: 0F
+            } else {
+                // Otherwise, we will select the desired star
+                desiredStopPoint
+            }
+        }
```

Comment:
I might be doing something wrong but that didn't seem to work for me. I could still select half a star.

<img width="287" alt="Screenshot 2024-07-04 at 11 35 42 AM" src="https://github.com/Automattic/pocket-casts-android/assets/308331/c13da43a-6070-4097-a0bf-5a69f2329be0">

Could another approach be to pass in the SpotPointType of `FullStars`?

<img width="767" alt="Screenshot 2024-07-04 at 11 34 32 AM" src="https://github.com/Automattic/pocket-casts-android/assets/308331/70dc1c9a-79f5-4d61-920c-24b2993375e7">

Reply:
@geekygecko  Sorry.  Maybe my PR description is not clear. The goal is to set the minimum rate to 1 star, which means you can't select zero or just one half star. This one you shared looks correct because you are trying to rate to 3,5 star 

So this only applies to the first star, that's why I had to check it